.travis.yml

```diff
@@ -1,23 +1,15 @@
 language: go
 sudo: false
 go:
-- 1.11.x
+  - 1.12.x
 services:
   - docker
 install:
   - chmod 777 -R "$(pwd)"
-  - make dep
-jobs:
-  include:
-    - stage: "Tests"
-      name: "Lint"
-      script:
-        - make lint
-        - make test
-        - make integration
-        - make coverage
-        - bash <(curl -s https://codecov.io/bash) -f .coverage/combined.cover.out
-    - stage: "Deploy"
```

Comment:
question: I assume since this was one of our earliest repos, the deploy step was included here because we didn't have the necessary Docker Hub hooks configured. In either case, have you confirmed that the docker image is still getting built?